components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -7057,6 +7059,50 @@ public static void deletePolicyPartialMappings(Integer policyGroupId, Connection
 		}
 	}
 
+    /**
+     * Remove XACML Policies from Entitlement Service
+     *
+     * @param policyGroupId
+     * @param conn
+     * @throws SQLException
+     */
+    public static void deleteXACMLPoliciesFromEntitlementService(Integer policyGroupId, Connection conn)
+            throws SQLException {
+        PreparedStatement ps = null;
+        ResultSet rs = null;
+        String query = " SELECT POLICY_ID FROM APM_POLICY_GRP_PARTIAL_MAPPING WHERE POLICY_GRP_ID=? ";
+        String policyId = "";
```

Comment:
Initialize to null instead of empty String. Following condition wont satisfy with this initialization .
if (policyId != null) {